### DISCUSSION TITLE
Updtae for GoPro Hero 11?

### DISCUSSION THREAD

**Discussion by 3DMiller** [KonradIT/gopro-py-api#207] [General]
Hello, are there planned updates for GoPro Hero 11 coming soon?

**Comment by robertgalo** [comment]
It would be great if there is an official GoPro 11 version out there. Generally I think, all features for GoPro 10 would fit for 11 too, or am I wrong?

**Reply by robertgalo** [reply]
Ps. We are trying to use OpenGoPro wired (through USB). Worked fine until we came to the point of controlling the files on the SD card. Currently we are not able to delete files. (Remark: the SD card serves in our use case as an cache only which has to be empty time by time)
